codeclimate-wrapper

```diff
@@ -41,17 +41,34 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+CC_XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config/}
+CC_XDG_CONFIG="${CC_XDG_CONFIG_HOME}/codeclimate/config.yml"
```

Comment:
Allowing the override does introduce and error case if users set a relative path. Something to consider.